pandas/compat/__init__.py

```diff
@@ -65,3 +65,16 @@ def is_platform_mac():
 
 def is_platform_32bit():
     return struct.calcsize("P") * 8 < 64
+
+
+def import_lzma():
+    import warnings
+    try:
+        import lzma
+        return lzma
+    except ImportError:
+        msg = (
+            "Could not import the lzma module. Your installed Python is incomplete. "
+            "Attempting to use `lzma` compression will result in a RuntimeError."
```

Comment:
How strong do we want to be in wording this warning message? I'm in favor of being quite explicit that the user's python is broken. So perhaps add something like "We recommend that you reinstall Python." though we can't really give guidance on what exactly went wrong with their previous install.

Reply:
Agree.
I am not sure if it would be too much to add, but if we know that this often happens due to a lack of `xz` being installed (on Mac, for example), we might want to mention that to the user.